src/hooks/useBalanceChartData/useBalanceChartData.ts

```diff
@@ -348,7 +349,19 @@ export const useBalanceChartData: UseBalanceChartData = args => {
   const cosmosCaip10 = account ? caip10.toCAIP10({ caip2: cosmosCaip2, account }) : ''
 
   // load staking data to redux state
-  useGetStakingDataQuery({ accountSpecifier: cosmosCaip10 })
+  useEffect(() => {
+    ;(async () => {
+      if (!cosmosCaip10?.length) return
+
+      dispatch(
+        stakingDataApi.endpoints.getStakingData.initiate(
+          { accountSpecifier: cosmosCaip10 },
+          { forceRefetch: true },
+        ),
+      )
+    })()
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
please include whatever is missing, or comment as to what was excluded and why

Reply:
Done, eslint was complaining about missing dispatch as a dependency, which we don't need